test/jdk/java/lang/Math/ExpCornerCaseTests.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2011,2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The year of the copyright syntax is "2011, 2020,"; no need for a re-review before pushing after that correction. Thanks.

Reply:
changed